core/src/main/java/org/mule/runtime/core/internal/processor/strategy/FluxSinkWrapper.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.core.internal.processor.strategy;
+
+import org.mule.runtime.core.api.event.CoreEvent;
+import reactor.core.Disposable;
+import reactor.core.publisher.FluxSink;
+import reactor.util.context.Context;
+
+import java.util.function.LongConsumer;
+
+public class FluxSinkWrapper implements FluxSink<CoreEvent> {
```

Comment:
Add a jdoc indicating that this is wrapper for verifying that the sink is being used. and a @see to the class where it is used.

Reply:
added java doc with @see